src/CodeGen_LLVM.cpp

```diff
@@ -603,6 +608,12 @@ void CodeGen_LLVM::begin_func(LoweredFunc::LinkageType linkage, const std::strin
     BasicBlock *block = BasicBlock::Create(*context, "entry", function);
     builder->SetInsertPoint(block);
 
+    // Put the global buffers in the symbol table.
+    for (const auto &buf : input_module->buffers()) {
+        llvm::Value *buf_ptr = sym_get(buf.name() + ".buffer");
```

Comment:
Still a WIP, but explicit_unpack_buffers deletes push_buffer/pop_buffer entirely, and also deletes the whole private __func hack. The only symbols that are in scope automatically are the .buffer ones (for globals and for args). The rest is done in IR.

Basically I wouldn't sweat naming decisions much here, because I'm going to try to make them consistent in the explicit_unpack_buffers PR.